src/api/app/models/bs_request_action_maintenance_incident.rb

```diff
@@ -185,7 +185,7 @@ def _merge_pkg_into_maintenance_incident(incident_project)
       new_pkg = Package.get_by_project_and_name(ret[:data][:targetproject], ret[:data][:targetpackage])
     elsif linkinfo && linkinfo['package'] # a new package for all targets
       if Package.exists_by_project_and_name(incident_project.name, source_package, follow_project_links: false)
-        new_pkg = Package.get_by_project_and_name(incident_project.name, source_package, use_source: false, follow_project_links: false)
```

Comment:
This change should be reverted since `follow_project_links: false` isn't a default option, so it should still be passed.

Reply:
ups